spec/lib/vet360/contact_information/service_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-describe Vet360::ContactInformation::Service do
+describe Vet360::ContactInformation::Service, skip_vet360: true do
```

Comment:
This disables stubbing of Vet360 so we can use our cassettes, but makes it so that Vet360 doesn't have to be setup for anyone else to use the User serializer